nncf/tensorflow/quantization/algorithm.py

```diff
@@ -362,11 +419,13 @@ def _get_quantizer_setup(self, model: tf.keras.Model) -> TFQuantizationSetup:
 
         quantizer_setup = self._get_quantizer_propagation_solution(nncf_graph,
                                                                    quantizable_weighted_layer_nodes,
-                                                                   custom_layer_nodes)
+                                                                   custom_layer_nodes,
+                                                                   model)
         setup = TFQuantizationSetup()
 
         quantized_layer_names_vs_qconfigs = {}  # type: Dict[str, QuantizerConfig]
-
+        qp_id_to_index = {} # type: Dict[QuantizationPointId, int]
+        index = 0
```

Comment:
Need a more verbose name for the `index` variable, describing just what index that variable represents.

Reply:
Renamed to `tf_setup_qp_index`